fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/sound/sound.xml

```diff
@@ -2354,6 +2354,11 @@
         <Sound WwiseData="true" WwiseEvent="fwdgalley" NodeName="SOUND_SPEAKER_LEFT" />
         <Sound WwiseData="true" WwiseEvent="cpdlc_ring" NodeName="SOUND_SPEAKER_LEFT" />
 
+        <!-- F/CTL Priority ====================================================================================-->
+        <Sound WwiseData="true" WwiseEvent="dualinput" NodeName="Wiper_Base_l" />
+        <Sound WwiseData="true" WwiseEvent="priorityleft" NodeName="Wiper_Base_l" />
+        <Sound WwiseData="true" WwiseEvent="priorityright" NodeName="Wiper_Base_l" />
```

Comment:
Don't use non SOUND_ nodes for sounds. We want to get away from that so the new model doesn't need to use old Asobo node names.